client/src/commands/connectCommand.ts

```diff
@@ -103,15 +104,20 @@ export async function connect(gatewayRegistryEntry: FabricGatewayRegistryEntry,
 
     // Get the identities
     const identityNames: string[] = await wallet.getIdentityNames();
+    const running: boolean = await FabricRuntimeManager.instance().getRuntime().isRunning();
+    if (!running) {
+        outputAdapter.log(LogType.ERROR, 'Please ensure your local fabric is running.');
```

Comment:
Can you make this 'local_fabric is not started, please start it before connecting' please?

Reply:
'local_fabric has not been started, please start it before connecting.'